src/legacy/server/config/schema.js

```diff
@@ -254,7 +254,8 @@ export default () =>
           )
           .default([]),
       }).default(),
-      manifestServiceUrl: Joi.string().default('https://catalogue.maps.elastic.co/v7.2/manifest'),
```

Comment:
Removing this will break kibana startup if users have this configured.

We should add a release-note so they are notified (regardless of the fact that this is an internal setting that was never publicly documented).

Reply:
This must be handled as a config deprecation that does not break Kibana startup. Config changes that prevent startup are considered breaking changes and a regression.

Now since this config is in the legacy "global config" rather than belonging to a specific plugin, I'm not sure the best place to put this. It seems you could register a ConfigDeprecationProvider here in the legacy bridge code: https://github.com/elastic/kibana/blob/master/src/core/server/legacy/legacy_service.ts#L146

@pgayvallet any suggestions? I thought we had a place in legacy for config deprecations for the global schema, but I can't seem to find it now.

Reply:
@joshdover that config-entry was never publicly documented and only used internally for debugging. Does it still need a deprecation warning?

Reply:
All core deprecations has been migrated to NP, there is no longer 'core legacy' deprecations. Best place to put it would be in NP in `src/core/server/config/deprecation/core_deprecations.ts`. It would be applied on both NP and legacy configuration

Reply:
Ah there it is, thanks @pgayvallet!